deposit_contract.sol

```diff
@@ -73,18 +73,18 @@ contract DepositContract is IDepositContract {
         bytes32 deposit_data_root
     ) override external payable {
         // Avoid overflowing the Merkle tree (and prevent edge case in computing `self.branch`)
-        require(deposit_count < MAX_DEPOSIT_COUNT);
+        require(deposit_count < MAX_DEPOSIT_COUNT, "DepositContract: merkle tree full");
```

Comment:
Are we suggesting to keep the revert strings in the final binary or not?

Reply:
I think its good to keep them in the binary!

Reply:
It helps in debuggability when doing integrations or lower level interactions with the contract